script.js

```diff
@@ -156,10 +156,23 @@ function searchVenues(places) {
 }
 
 // Adds a marker to the map and push to the array.
-function addMarker(latLng, id) {
+function addMarker(latLng, id, number) {
+  var colorGradient = ['#F086A2','#DE809F','#CD7B9D','#BC759A','#AB7098','#9A6B96','#896593','#786091','#58568C','#45508A'];
+  var icon = {
```

Comment:
for icons do you need  `var iconBase = 'https://maps.google.com/mapfiles/kml/shapes/` in your code?